src/stdlib/native/multiplexer.no

```diff
@@ -0,0 +1,43 @@
+use std::comparator;
+
+fn escalar_product(lhs: [Field; LEN], rhs: [Field; LEN]) -> Field {
```

Comment:
BTW let's add some documentation for each of these functions! Let's not replicate the no-doc land of circomlib :D